plugin/evm/header/block_gas_cost.go

```diff
@@ -69,37 +69,40 @@ func BlockGasCostWithStep(
 	)
 }
 
-// MinRequiredTip is the estimated minimum tip a transaction would have
-// needed to pay to be included in a given block (assuming it paid a tip
-// proportional to its gas usage). In reality, there is no minimum tip that
-// is enforced by the consensus engine and high tip paying transactions can
-// subsidize the inclusion of low tip paying transactions. The only
-// correctness check performed is that the sum of all tips is >= the
-// required block fee.
+// EstimateRequiredTip is the estimated tip a transaction would have needed to
+// pay to be included in a given block (assuming it paid a tip proportional to
+// its gas usage).
+//
+// In reality, there is no minimum tip that is enforced by the consensus engine
```

Comment:
nit: active language :)
```suggestion
// In reality, the consensus engine does not enforce a minimum tip
```